rero_ils/modules/patrons/api.py

```diff
@@ -775,6 +779,52 @@ def get_current_patron(record):
             if ptrn.organisation_pid == record.organisation_pid:
                 return ptrn
 
+    @classmethod
+    def set_communication_channel(
```

Comment:
maybe the `@setter` decorator could be better than a class method ?
An example :

```python
class Geeks:
     def __init__(self):
          self._age = 0
       
     # using property decorator
     # a getter function
     @property
     def age(self):
         print("getter method called")
         return self._age
       
     # a setter function
     @age.setter
     def age(self, a):
         if(a < 18):
            raise ValueError("Sorry you age is below eligibility criteria")
         print("setter method called")
         self._age = a
```